frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -248,3 +249,16 @@ export const usePrometheusPoll: UsePrometheusPoll = (props) => {
   // unify order with the rest of API
   return [result[0], !result[2], result[1]];
 };
+
+/**
+ * A component to render timestamp.
+ * The timestamps are synchronized between invidual instances of the Timestamp component.
+ * The provided timestamp is formatted according to user locale.
+ *
+ * @prop timestamp - the timestamp to render
```

Comment:
We should discuss what formats are accepted

Reply:
Ok, I added 
```
The timestamp format should conform to any form accepted by the new Date() constructor
```
Not sure it that is fine, as it describes internal implementation of the Timestamp component, but Im not sure how to better describe the accepted formats.

Reply:
I would say we support ISO 8601 dates as used by Kubernetes or epoch timestamps as a number.

https://tc39.es/ecma262/#sec-date-time-string-format

Reply:
updated